trainers/datasets.py

```
@@ -56,15 +56,6 @@ def __len__(self):
     
     def __getitem__(self, idx):
         raise NotImplementedError
-    
-    def check_processed(self):
```

doesn't do anything...

---

I think it is worth keeping. When people init the dataset without it being preprocessed we should throw an error.

---

it literally isn't called anywhere. fine to add it into the init but rn it doesn't do anything